service/install-service.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+cp service/threeal-bot.service /lib/systemd/system
```

Comment:
Let's make this more interactive, first check if `/lib/systemd/system/threeal-bot.service` is exist or not.

Just output something if it's exist, e.g.:
```bash
Foo is already installed
```

Else, output an install notice, copy the config file, and output a success notice.
```bash
Installing Foo...
# do copy
Done installing Foo
```